dap_prinz_green_jobs/pipeline/green_measures/skills/README.md

```diff
@@ -1,62 +1,115 @@
-# Green Skills
+# 🧑‍🔧🥬 Green Skills
 
-This directory contains the scripts needed to extract green skills for a given job advert.
+This directory contains the scripts needed to extract green skills for a given job advert. This directory is organised as follows:
 
-The script
+1. [skills_measures_utils.py](https://github.com/nestauk/dap_prinz_green_jobs/tree/dev/dap_prinz_green_jobs/pipeline/green_measures/skills/skills_measures_utils.py): The script which contains the `SkillMeasures` class to find green skills in job adverts. This utilises our [Skills Extractor library](https://github.com/nestauk/ojd_daps_skills) to extract and map skills onto a structured green skills list.
+2. [green_skill_classifier.py](https://github.com/nestauk/dap_prinz_green_jobs/tree/dev/dap_prinz_green_jobs/pipeline/green_measures/skills/green_skill_classifier.py): A script to train and use a model to predict whether a skill is green or not.
+3. [evaluation/](https://github.com/nestauk/dap_prinz_green_jobs/tree/dev/dap_prinz_green_jobs/pipeline/green_measures/skills/evaluation/): A directory with the evaluation of how well the `SkillMeasures` class extracts and maps skills.
 
-```
-dap_prinz_green_jobs/pipeline/green_measures/skills/skills_measures_utils.py
-```
+It also contains the following `.py` files:
 
-contains the main functions needed to extract green skills. As a baseline, we are using the `custom usage` feature of the Skills Extractor library to extract and map skills onto a structured green skills list.
+1. [green_esco_formatting.py](https://github.com/nestauk/dap_prinz_green_jobs/tree/dev/dap_prinz_green_jobs/pipeline/green_measures/skills/green_esco_formatting.py): A script to format the ESCO green skills taxonomy into a structure compatible with the Skills Extractor library. This needs to be run as a one-off.
+2. [map_skills_utils.py](https://github.com/nestauk/dap_prinz_green_jobs/tree/dev/dap_prinz_green_jobs/pipeline/green_measures/skills/map_skills_utils.py): Utils to match skill embeddings to all of the ESCO taxonomy. The majority of this code is from the [ojd_daps_skills](https://github.com/nestauk/ojd_daps_skills) repo, but replicated here since currently there was no functionality in `ojd_daps_skills` to take in already embedded skills and map them.
+3. [customise_skills_extractor.py](https://github.com/nestauk/dap_prinz_green_jobs/tree/dev/dap_prinz_green_jobs/pipeline/green_measures/skills/customise_skills_extractor.py): To add the custom config file, formatted skills taxonomy and taxonomy embeddings to the relevant location in the `ojd-daps-skills` library location run this script with the `--config_name "extract_green_skills_esco"` argument. NOTE: it isn't neccessary to run this anymore, as such this file is unneeded.
```

Comment:
hmmm should we just delete this file then?

Reply:
yeh I guess so .. I suppose its a bit confusing to keep it